server/routes/api/configuration.js

```diff
@@ -1,13 +1,35 @@
 var router = require('express').Router(),
-    User = require('../../db/model/User.js');
+    ConfigurationSchema = require('../../db/model/Configuration.js'),
+    authMiddleware = require('../../middleware/auth.js'),
+    Responses = require('../../responses/api/announcement.js');
 
 // Handles get requests for /v1/configuration
 router.get('/', function(req, res) {
-    const configuration = {
-        is_livepage_enabled: false,
-        is_applications_open: true
-    };
+    ConfigurationSchema.findOne({}, '-_id -__v')
+        .exec()
+        .then(configuration => {
+            if (configuration) {
+                res.send({
+                    status: true,
+                    configuration: configuration
+                });
+            } else {
+                res.send({
+                    status: false,
+                    configuration: {}
+                });
+            }
+        })
+        .catch(err => {
+            console.error(err);
+            res.send({
+                status: false,
+                message: Responses.UNKNOWN_ERROR
+            });
+        });
 
+    //TODO I don't know what this did so I didn't remove it yet
+    /*
     if (req.get('Authorization')) {
```

Comment:
I'm gonna guess this was for getting the configuration with a user object. Let's keep them as separate endpoints